Berksfile.lock

```diff
@@ -0,0 +1,42 @@
+DEPENDENCIES
```

Comment:
This is to lock down chef recipe dependencies so we don't get any surprises. This was ignored by default by the cookbook generator, which seems very, very wrong.